Earth disappears when adding "paleo_map.asset"

(as reported on slack)

`Error linking program object [LocalChunkedLodPatch]: Fragment info
-------------
(0) : error C6020: Constant register limit exceeded at lightDirectionCameraSpace; more than 1024 registers needed to compile program
(0) : error C6020: Constant register limit exceeded at opacity; more than 1024 registers needed to compile program
`
This is when loading
scene/solarsystem/planets/earth/noaa-sos/land/paleo_map
in v0.21.2 


Seems that it's enabling too many layers

--- a/data/assets/scene/solarsystem/planets/earth/noaa-sos/land/paleo_map.asset
+++ b/data/assets/scene/solarsystem/planets/earth/noaa-sos/land/paleo_map.asset
@@ -127,7 +127,7 @@ for i,v in ipairs(layers_names) do
   local layer = {
     Identifier = Identifier .. "-" .. v,
     Name = Name .. " " .. v,
-    Enabled = asset.enabled,
+    Enabled = false,
     ZIndex = 100,
     FilePath = imagesDestination .. "/" .. v .. ".jpg",
     Description = Description

